includes/admin/class-sensei-setup-wizard-pages.php

```diff
@@ -99,4 +99,40 @@ private function get_learner_courses_page_content() {
 		return implode( $blocks );
 	}
 
+	/**
+	 * Get the block content for course completed.
+	 *
+	 * @return string
+	 */
+	private function get_course_completed_page_content() {
+		$blocks   = [];
+		$blocks[] = serialize_block(
+			[
+				'blockName'    => 'core/paragraph',
+				'innerContent' => [ '<p class="has-text-align-center">' . __( 'Congratulations on completing the course!', 'sensei-lms' ) . '</p>' ],
```

Comment:
Will it be possible to add the party popper emoji here?

Reply:
https://github.com/Automattic/sensei/pull/4281/commits/1cb2843d43941f3a1ab22f80640ae9ceb453cbcf 🥳

I tried adding the unicode, but the editor complained. I think adding the emoji directly is okay!